README.md

```diff
@@ -54,5 +54,31 @@ will begin to drift slightly from the original design. This is expected and
 there is no expectation that the original design proposal needs to be updated
 to reflect these differences.
 
-The code and our documentation are the ultimate sources of truth. CFPs are merely
-the starting point for the implementation.
+The code and our documentation are the ultimate sources of truth. CFPs are
+the starting point for the implementation, but can also be updated as needed.
+
+# Status
+
+The status of a CFP indicates its maturity. There are four different statuses
+that a CFP can have:
+
+1. Draft: A CFP in any form (Google doc, Github issue, ect.) that has not yet
+been merged into this repository.
+
+2. Implementable: CFPs that have been approved by one committer and merged into
+this repository are given the implementable status. This means that the ideas in the CFP
+have been agreed in principle and the coding work can now be started. When merged,
+the CFP should be up to date and the relevant stakeholders should be in alignment
+even if they are still going through the practical ramifications of how to implement it.
+
+3. Released [Project] X.Y: Everything listed in the CFP as part of the proposal (not future milestones) is merged into the project repository with the listed release.
+
+4. Dormant: This status is given to a CFP that has not been
```

Comment:
How much effort should we expect authors of dormant CFPs to put into clarifying the status of them?

One of the goals around "Dormant" status is I think to capture ideas that were considered but are not currently a priority. Sometimes these proposals may have aspects which are not agreed upon. While I can see the use of merging a record of the discussion into the repository, I worry a little that we might merge a dormant CFP with aspects that have no consensus, then subsequently someone may pick up the dormant PR and not address the concerns from those aspects with no consensus. The resulting PR might just look like a general tidy-up / improvement of a previous CFP.

Ultimately I guess this would just be up to the rigor of the review for when the status is moved to "Implementable", maybe the answer is that simple - make sure the full CFP has been re-reviewed prior to changing the status.

Reply:
@bowei this is related to [your thread above](https://github.com/cilium/design-cfps/pull/42#discussion_r1702120588), would be curious about your thoughts on this.

Reply:
It looks like at least https://github.com/cilium/design-cfps/pull/7 and https://github.com/cilium/design-cfps/pull/36 are in this category.

Reply:
Yeah, I think this is a responsibility for the flip to "implementable" -- don't just do it without clear signal the proposal is agreed upon. This is something that would be easy to resolve without added procedure -- it should be hard to have a submarine proposal and anything that wasn't supposed to be merged would probably be backed out at some point with the biggest loss being work...

Reply:
Agree, I think that is part of the switch to implementable and just requires people to be thoughtful in their reviews